src/reporters/lang/en.js

```diff
@@ -257,6 +256,7 @@ const messages = {
   optionalDepNotInstalled: 'Optional dependency $0 not installed',
   packageWrongVersion: '$0 is wrong version: expected $1, got $2',
   packageDontSatisfy: '$0 doesn\'t satisfy found match of $1',
+  packageContainsYarnAsGlobal: 'installing Yarn via Yarn will result in you having two separate versions of Yarn installed at the same time.',
```

Comment:
Maybe also mention that it's not recommended, and to install Yarn by following the instructions at https://yarnpkg.com/en/docs/install?